src/NearOrg/DataAvailabilityPage.jsx

```diff
@@ -323,9 +323,9 @@ return (
                 <Widget
                   src="${REPL_ACCOUNT}/widget/DIG.Button"
                   props={{
-                    href: "https://forms.gle/LWJoTpTiCbSheUL26",
+                    href: "https://${REPL_NEAR_URL}/blog/why-near-data-availability",
```

Comment:
@charleslavon are we good to use near.org here instead of pages.near.org?

Reply:
yes